docs-ref-conceptual/azure-cli-configuration.md

```diff
@@ -35,21 +35,21 @@ The following table contains a list of available configuration keys.
 |------|-------------|
 | group | The default resource group to use for all commands. |
 | location | The default location to use for all commands. |
-| web | The default app name to use for **az webapp** commands. |
-| vm | The default VM name to use for **az vm** commands. |
-| vmss | The default virtual machine scale set (VMSS) name to use for  **az vmss** commands. |
-| acr | The default container registry name to use for **az acr** commands. |
+| web | The default app name to use for `az webapp` commands. |
+| vm | The default VM name to use for `az vm` commands. |
+| vmss | The default virtual machine scale set (VMSS) name to use for  `az vmss` commands. |
+| acr | The default container registry name to use for `az acr` commands. |
 
 As an example, here's how you would set the default resource group and location for all commands.
 
-`azurecli-interactive
+```azurecli-interactive
 az configure --defaults location=westus2 group=MyResourceGroup
-`
```

Comment:
This will cause incorrect rendering

![image](https://user-images.githubusercontent.com/4003950/103724915-6c345580-5010-11eb-84b6-d109c9e4c8fc.png)

Reply:
Splitting this issue to a separate PR https://github.com/MicrosoftDocs/azure-docs-cli/pull/2377 while this one is under debate.